Library/Homebrew/test/cli/parser_spec.rb

```diff
@@ -115,6 +115,26 @@
     end
   end
 
+  describe "test comma array options" do
+    subject(:parser) {
+      described_class.new do
+        comma_array "--repos", "--repositories=", description: "List of repositories"
```

Comment:
```suggestion
        comma_array "--repos=", "--repositories=", description: "List of repositories"
```

Reply:
Should these be consistent? (They don't have `=` in the `contributions` definition, and not all of the `comma_array` usages have `=` either.) Based on [the flags code](https://github.com/Homebrew/brew/blob/044fefdce91ce1a4dde8945ae6292dce74a0258f/Library/Homebrew/cli/parser.rb#L208-L212) it would seem like the `=` is a marker for required or not, but for `comma_array` [every one is required](https://github.com/Homebrew/brew/blob/044fefdce91ce1a4dde8945ae6292dce74a0258f/Library/Homebrew/cli/parser.rb#L202), hence why I'm not sure. 🤔